src/pages/about.md

```diff
@@ -1,13 +1,77 @@
-## PingCAP
+---
+title: Hello, Open Source Software Insight
+date: 2022-05-05
+image: https://twitter.com/OSSInsight/photo
+authors: [OSSInsight]
```

Comment:
It's not a /blog page, so the authors field won't be rendered.

Reply:
ok just delete it ?

> It's not a /blog page, so the authors field won't be rendered.